WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -122,6 +122,37 @@
         });
       </script>
 
+
+      <% if @competition_admin_view %>
```

Comment:
select datediff('0-0-0', '0-0-0') <= 90
also gives NULL

and so does select datediff('0-0-0', '2015-11-10') <= 90

Not sure what you mean by writing a test...these tests seem conclusive to me.